### DISCUSSION TITLE
Make any part of the window transparent (and apply blur on it).

### DISCUSSION THREAD

**Discussion by enzobelmont** [tsujan/Kvantum#820] [Q&A] [answered]
Hi, sorry to be annoying, but i've read all kvantum documentation and I can not find the correct parameter to achieve transparency in the whole applications window, just like pcman-fm and dolphin. 

I would like to have a fully blurred qbittorrent window (or maybe elisa, kdiskfree
, even systemsettings) with a glassy-kv theme on it (i.e.). I've modified several themes to create a mine own, but I fail to get a fully blurred window.

can you throw me an advice to achieve it?

thanks in advance.

![Screenshot_20220823_074502](https://user-images.githubusercontent.com/72204911/186162301-98dffd75-102a-467d-8bcf-4ea1743548da.png)

**Comment by tsujan** [comment, marked as answer]
Make `base.color` and `alt.base.color` translucent *cautiously*, by choosing appropriate 8-digit hex numbers, so that if you remove the last two digits, the remaining 6-digit hex numbers are acceptable colors. Never use totally transparent colors for them.

![base_olor](https://user-images.githubusercontent.com/981076/186174571-55395890-0094-4d27-b4b3-70b9fb82e041.png)

This is not recommended but works with most apps. Kvantum may need to make those colors opaque in some parts of some apps again, for various reasons.
